src/assets/images.js

```diff
@@ -0,0 +1,9 @@
+const images = {
+    camera: require('./images/camera.png'),
```

Comment:
can we not use the svg versions? We should if React native supports them

Reply:
Svg's don't work directly. There are some packages that claim to work with them. I chose to convert to png so I could get the basic home page complete. I could revisit enabling svg support in a separate PR?